ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientImpl.java

```diff
@@ -355,7 +382,17 @@ public void close() {
       final T cf,
```

Comment:
Let's rename these existing `makeRequest()` methods `makePostRequest()` to clarify. Up to you whether you want the new one you added to be `makeRequest()` or `makeGetRequest()` (and hard-code `HttpMethod.GET`).